integrations/src/test/java/io/jenkins/plugins/casc/GiteaOrganisationFolderTest.java

```diff
@@ -0,0 +1,69 @@
+package io.jenkins.plugins.casc;
+
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import java.util.List;
+import jenkins.branch.OrganizationFolder;
+import jenkins.scm.api.trait.SCMTrait;
+import org.jenkinsci.plugin.gitea.BranchDiscoveryTrait;
+import org.jenkinsci.plugin.gitea.ExcludeArchivedRepositoriesTrait;
+import org.jenkinsci.plugin.gitea.ForkPullRequestDiscoveryTrait;
+import org.jenkinsci.plugin.gitea.GiteaSCMNavigator;
+import org.jenkinsci.plugin.gitea.OriginPullRequestDiscoveryTrait;
+import org.jenkinsci.plugin.gitea.SSHCheckoutTrait;
+import org.jenkinsci.plugin.gitea.TagDiscoveryTrait;
+import org.jenkinsci.plugin.gitea.WebhookRegistrationTrait;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.recipes.WithPlugin;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class GiteaOrganisationFolderTest {
+
+    @Rule
+    public JenkinsConfiguredWithCodeRule r = new JenkinsConfiguredWithCodeRule();
+
+    @Test
+    @WithPlugin({"branch-api.jpi", "cloudbees-folder.jpi", "scm-api.jpi", "structs.jpi"})
```

Comment:
I don't see any reason to add the binary of these plugins, just include them as test dependencies (which I believe most of the mare) in the pom of the integrations project

Reply:
Only scm api plugin is there but the rests are properly transitive dependencies already.
https://github.com/jenkinsci/configuration-as-code-plugin/blob/faba0f7d0cf8e34015199edeae2a3e1270ee20f9/integrations/pom.xml#L291-L295

Reply:
See the reason here: https://github.com/jenkinsci/configuration-as-code-plugin/pull/1791#discussion_r762444212

Reply:
For some reason GitHub cannot properly link to the comment.

Reply:
its in a resolved conversation which breaks linking i've unresolved